notes-app/app.js

```diff
@@ -0,0 +1,71 @@
+// npm packages:
+const yargs = require('yargs');
+
+// custom packages:
+const noteHelper = require('./notes.js')
+
+// Create add command
+yargs.command({
+    command: 'add',
+    describe: 'Add a new note.',
+    builder: {
+        title: {
+            describe: 'Title of the note being added.',
+            demandOption: true,
+            type: 'string'
+        },
+        body: {
+            describe: 'Body of the note being added.',
+            demandOption: true,
+            type: 'string'
+        }
+    },
+    handler(argv) {
+        noteHelper.addNote(argv.title, argv.body);
+    }
+});
+
+// Create remove command
+yargs.command({
+    command: 'remove',
+    describe: 'Remove a note.',
+    builder: {
+        title: {
+            describe: 'Title of note to be removed.',
```

Comment:
use a common tense across all sentences/phrases. 
add contains "note being added"
remove contains "note to be removed"
Though grammatically they are perfect, I believe using a common tense would maintain uniformity.
example:
add = "Note to be added"
remove = "Note to be removed"
...

Reply:
done

Reply:
done